File: features/class_declaration/person_class.swift
class Person {
    var name : String
    init(name: String) {
       self.name = name
    }
    
	func present() -> String {
        return "Yes"
    }
	
	func greet(other: String) -> String { 
		return other 
	}
}


val A = Person("A");
val B = Person("B");

File: features/class_declaration/person_class.cpp
class Person {
    // Access specifier
    public:
 
    // Data Members
    string name;
 
    // Member Functions()
    string present() {
       return "Yes";
    }
	
	string greet(string other) {
		return other;
	}
};
 
int main() { 
    // Declare an object of class geeks
    Person a;
	Person b;
 
    // accessing data member
    a.name = "A";
 	b.name = "B";
	
    return 0;
}